test/libsolidity/syntaxTests/nameAndTypeResolution/440_warn_unused_return_parameter_with_explicit_return.sol

```diff
@@ -1,7 +0,0 @@
-contract C {
-    function f() pure public returns (uint a) {
-        return;
```

Comment:
I assume we have a test here that is similar to this one, but has `return 1` here?

Reply:
Yes, https://github.com/ethereum/solidity/blob/8e93b729a4fb50d1d0f21b86a7434ed91ae8826a/test/libsolidity/syntaxTests/nameAndTypeResolution/444_unnamed_return_parameter_with_explicit_return.sol#L3

However, we don't warn in this case and I wonder, why not and whether we should change that.

Reply:
That would be a separate issue, though.